docs/help/contributor/style-guide/documenting-code.md

```diff
@@ -0,0 +1,212 @@
+# Documenting Code
+
+??? tip "Looking for a template?"
+    //todo
+
+## Documenting Commands
+
+>**The standard line is “X the Y by running:”**
+
+>It meets these criteria:
+
+>* Explicitly mentions running the command (this isn’t always obvious)
+* **Uses “run”** (and not “type”, “execute”, etc -- we want consistency)
+* Starts with the key information that describes the command, e.g. “To do X...enter Y command:”, “Do X by entering Y command:”
+* As short as possible
+
+> If you must deviate from the standard line, ensure you still meet the above criteria.
+
+=== ":white_check_mark: Correct"
+    Create the service by running:
```

Comment:
```suggestion
    Create a service by running the command:
```

Reply:
Resolved this before I saw it here!

Reply:
I left out "following" for brevity, happy to revisit if you feel it's important